cbv3_django_prototype/cbv3_django_prototype/users/fixtures/users.json

```diff
@@ -6,7 +6,7 @@
 "password":"argon2$argon2i$v=19$m=512,t=2,p=2$NkticWRCd3VCaXNR$SaNtr1/gOOSbzxfBDtAcpw",
 "last_login":"2020-01-07T02:32:09.518Z",
 "is_superuser":false,
-"username":"bethanyg",
+"username":"bethanygg",
 "first_name":"",
 "last_name":"",
 "email":"gbethanyg@gmail.com",
```

Comment:
Yup.  Will do - thanks for checking.  That's a spam trap, but no need to have my arse hanging out in any fashion.  ;-)  Will be re-editing and pushing fixture changes.

Reply:
**Additional note:** whilst addressing this issue, realized that when `resources.json` is loaded in the test DB **new guids are being generated**.  In fact, new guids are are also being generated whenever a POST is sent - even if a guid is **included**.  I think this needs to be logged as a separate bug/issue and discussed.  One of the  features of a guid is that it can be generated by the client....but if it is being overwritten on the back end, it defeats this purpose.